tools/alignment_tools.py

```diff
@@ -0,0 +1,186 @@
+        elif self.alignments_format == 'p':
+            return pickle.dumps(self.alignments)
+        elif self.alignments_format in ('yaml', 'yml'):
+            try:
+                import yaml
+            except ImportError:
+                print("You must have PyYAML installed to use YAMLSerializer")
+            return yaml.dump(self.alignments, default_flow_style=False)       
+
+    def save_alignments(self):
+        ''' Backup copy of old alignments and save new alignments '''
+        self.backup_alignments()
+        alignments = open(self.alignments_file, 'w')
+        alignments.write(self.write_alignments())
+        alignments.close()
+
+class Faces(object):
+    ''' Object to hold the faces that are to be swapped out '''
+    def __init__(self, faces_dir):
+        self.faces_dir = faces_dir
+        self.faces = {}
+
+        self.load_faces()
+
+    def check_folder_exists(self):
+        ''' makes sure that the faces folder exists '''
+        if not os.path.isdir(self.faces_dir):
+            raise ValueError ('The folder {} could not be found'.format(self.faces_dir))
+
+    def load_faces(self):
+        ''' Load the face names into list '''
+        print('Loading faces from {}'.format(self.faces_dir))
+        try:
+            self.check_folder_exists()
+            for face in os.listdir(self.faces_dir):
+                face_filename = os.path.splitext(face)[0]
+                index = int(face_filename[face_filename.rindex('_')+1:])
+                original_file = '{}{}'.format(face_filename[:face_filename.rindex('_')], os.path.splitext(face)[1])
+                if self.faces.get(original_file, '') == '':
+                    self.faces[original_file] = [index]
+                else:
+                    self.faces[original_file].append(index)
+        except ValueError as v:
+            print(v)
+            sys.exit()
+        except:
+            raise
+
+def bad_args(args):
+    parser.print_help()
+    exit(0)
+
+def parser_arguments(parser):
+    parser.add_argument('-j', '--job',
+                        type=str,
+                        choices=('remove', 'merge', 'reformat'),
+                        dest='job',
+                        help='The job to be performed.',
+                        required=True)
+    parser.add_argument('-a', '--alignments_folder',
+                        type=str,
+                        dest='alignments_dir',
+                        help='Input directory of source A frames. Alignments file must exist in '
+                              'this folder',
+                        required=True)
+    parser.add_argument('-f', '--faces_folder',
+                        type=str,
+                        dest='faces_dir',
+                        help='Input directory of source A extracted faces.',
+                        required=True)
+
+def remove_alignments(arguments):
+    ''' Remove items from alignments file '''
+    faces = Faces(arguments.faces_dir)
```

Comment:
This is minor, but to avoid having to do `faces.faces` later on, you can just do:
```python
faces = Faces(arguments.faces_dir).faces
```

Reply:
Yeah, it's not perfect, but I'm going to leave as is, as I plan to (and have) added more variables.